Dockerfiles/compliance-operator-content-konflux.Containerfile

```diff
@@ -94,7 +94,7 @@ RUN grep -lr 'documentation_complete: false' ./products | xargs -I '{}' \
 
 # Build the OpenShift and RHCOS content for x86, aarch64 and ppc64le architectures.
 # Only build OpenShift content for s390x architectures.
-RUN if [ "$(uname -m)" = "x86_64" ] || [ "$(uname -m)" = "aarch64" ] || [ "$(uname -m)" = "ppc64le"]; then \
+RUN if [ "$(uname -m)" = "x86_64" ] || [ "$(uname -m)" = "aarch64" ] || [ "$(uname -m)" = "ppc64le" ]; then \
```

Comment:
Would this have been caught by a linter?

Reply:
Doesn't look like this was needed upstream - https://github.com/ComplianceAsCode/content/blob/master/Dockerfiles/compliance-operator-content-konflux.Containerfile#L76C59-L76C66